src/clarity.ts

```diff
@@ -104,10 +104,19 @@ export function tag(key: string, value: string): void {
 export function upgrade(key: string): void {
   // Do not process upgrade call if Clarity is not already activated and in lean mode
   if (status && configuration.lean) {
+    configuration.lean = false;
     measure(data.upgrade)(key);
   }
 }
 
+export function consent(): void {
+  // Do not begin tracking user if Clarity is not already activated
+  if (status && !configuration.track) {
+    configuration.track = true;
+    measure(data.track)();
```

Comment:
>track [](start = 17, length = 5)

Could we call this consent to make it clearer? Track is vague.

Reply:
Sure, I will implement the change.

Reply:
On second thoughts after making the change I think the previous version was better. Because we do expose consent call to the user of the API. Once it's called, we update the config and then  start tracking the user. If I renamed the function to consent, it wouldn't make much sense in the default case where we use it to track the user.

Reply:
It's called consent() in latest version. Can you change it t a cleaere name than just "track" if consent is not clear? For example enableTracking() or startTracking() or something. Your concise names are sometimes a bit cryptic. :-)

---
In reply to: [396137009](https://github.com/microsoft/clarity-js/pull/170#discussion_r396137009) [](ancestors = 396137009)